config/software/nghttp2.rb

```diff
@@ -1,10 +1,10 @@
 name "nghttp2"
-default_version "1.12.0"
+default_version "1.41.0"
 
 dependency "openssl"
 
-source url: "https://github.com/nghttp2/nghttp2/releases/download/v1.12.0/nghttp2-#{version}.tar.gz",
```

Comment:
Could we keep the previous md5sum ? It make reverting version much easier.

See [this](https://github.com/DataDog/omnibus-software/blob/master/config/software/libtool.rb#L26) or [this](https://github.com/DataDog/omnibus-software/blob/master/config/software/libpng.rb#L23) for examples.

Reply:
Thanks @hush-hush for the suggestion !
I’ve implemented it.